src/Spago/Command/Ls.purs

```diff
@@ -45,6 +53,18 @@ type LsEnv =
   , selected :: WorkspacePackage
   }
 
+listPaths :: Spago { logOptions :: LogOptions } Unit
+listPaths = do
+  logDebug "Running `listPaths`"
+  traverse_ logInfo
```

Comment:
`logInfo` goes to stderr, which we use for logging. Program output should go to `stdout`.

We already have facilities for this (the `output` function), see e.g. how the other `ls` commands or the `registry‘ commands do it.
It's quite fancy at this point 😄

We might want a table here, as it's the default output mode for the other ls commands too